docs/build/abci++/00-introduction.md

```diff
@@ -0,0 +1,52 @@
+# Introduction
+
+## What is ABCI++?
```

Comment:
```suggestion
## What is ABCI?
```
any chance we can call abci++ just abci? it will get confusing in the future if abci 3.0 comes out. Is it abci+++ or abci 3.0.

Reply:
+1